fem/fespace.hpp

```diff
@@ -571,7 +571,7 @@ class FiniteElementSpace
 
    /** @brief Returns pointer to the FiniteElement in the FiniteElementCollection
         associated with i'th element in the mesh object. */
-   const FiniteElement *GetFE(int i) const;
+   virtual const FiniteElement *GetFE(int i) const;
```

Comment:
I think we don't have to touch anything in the `(Par)FiniteElementSpace` in this PR. Just like with `GridFunction::GetElementDofValues`, we can have `GridFunction::GetFE` that does the hiding of the difference between regular/neighbor elements.